app/models/katello/trace_status.rb

```diff
@@ -35,9 +35,12 @@ def to_global(_options = {})
     end
 
     def to_status(_options = {})
-      if host.host_traces.reboot_required.any?
+      traces = host.host_traces.pluck(:app_type).uniq
```

Comment:
I wonder if calculating the unique set of items is worth it. I think it just adds overhead, but I don't have any idea if that's actually true.